src/components/single-line-diagram.js

```diff
@@ -77,9 +81,11 @@ const SingleLineDiagram = (props) => {
                     setSvg({svg: null, metadata: null, error, svgUrl: props.svgUrl});
                 });
         } else {
-            setSvg(nosvg);
+            setSvg(noSvg);
         }
-    }, [props.svgUrl]);
+    }, [props.svgUrl, useSelector(state => state.currentSvg)]);
```

Comment:
Hmm a dependency that is not used ? Is there no other way ?

Reply:
Ah I see, then It's better to use a imperative handle to force the refresh of the svg I think